<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>人员管理系统</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="2,2,20,1">
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>41</number>
           </property>
           <item>
            <widget class="QTableView" name="m_tvUserInfo"/>
           </item>
           <item>
            <widget class="QPushButton" name="m_btnSync">
             <property name="text">
              <string>同步&gt;&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="m_tvMenJinInfo"/>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1,1,20">
              <item row="0" column="0">
               <widget class="QPushButton" name="m_btnAddUser">
                <property name="text">
                 <string>添加</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="m_btnChangeUser">
                <property name="text">
                 <string>修改</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="m_btnDelUser">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>用户信息</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>门禁信息</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
